strawberry_django/mutations/resolvers.py

```diff
@@ -351,7 +448,8 @@ def update(
         for field, value in m2m:
             update_m2m(info, instance, field, value)
 
-        retval.append(instance.__class__._default_manager.get(pk=instance.pk))
+        instance.refresh_from_db()
```

Comment:
If I remember correctly, the reason I decided to retrieve the object again instead of refreshing it from the db was to make sure that one-to-many and many-to-many relations would be refetched as well.

Wondering if we should do `instance._prefetched_objects_cache.clear()` and `instance._state.fields_cache.clear()` as well. Maybe it is not required though

Reply:
Shouldn't refresh_from_db() do that as well?  
I see in other projects, that if I have out-of-date info through FK and M2M - adding refresh_from_db() resolves that without having to refetch.  

There is in my experience only one case where I need to refetch in a similar way to here.  That's in combination with Django-polymorphic. That that's due to the app bungling up the content types.  That's not the case here.

Reply:
Hrm, you are correct!

I remember having to do that in another project, but maybe it is not required anymore :)

Considering this as resolved